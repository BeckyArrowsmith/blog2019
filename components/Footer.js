import Link from "next/link";

const Footer = () => (
    <footer>
        <div className="container">
            <Link href="/about">
                <a>About</a>
            </Link>
            <Link href="http://github.com/BeckyArrowsmith">
                <a>GitHub</a>
            </Link>

            <Link href="https://github.com/BeckyArrowsmith/blog2019">
                <a>Website repo</a>
            </Link>

            <Link href="/pgp.txt">
                <a>PGP key</a>
            </Link>

            <p>&copy; Becky Arrowsmith {new Date().getFullYear()} &#128029;</p>
        </div>
    </footer>
);

export default Footer;