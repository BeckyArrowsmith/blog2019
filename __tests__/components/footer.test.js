/* eslint-env jest */

import { shallow } from 'enzyme'
import React from 'react'
import renderer from 'react-test-renderer'

import App from '../../pages/index.js'
import Footer from '../../components/Footer.js'

describe('With Enzyme', () => {
    it('Footer renders without crashing', () => {
      shallow(<Footer />);
    });
})

describe('With Snapshot Testing', () => {
    it('Footer renders without crashing', () => {
        const component = renderer.create(<Footer />)
        const tree = component.toJSON()
        expect(tree).toMatchSnapshot()
    })
})