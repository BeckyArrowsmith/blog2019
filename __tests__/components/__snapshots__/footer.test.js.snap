// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`With Snapshot Testing Footer renders without crashing 1`] = `
<footer>
  <a
    href="/"
    onClick={[Function]}
    onMouseEnter={[Function]}
    style={
      Object {
        "marginRight": 15,
      }
    }
  >
    link
  </a>
  <a
    href="/about"
    onClick={[Function]}
    onMouseEnter={[Function]}
    style={
      Object {
        "marginRight": 15,
      }
    }
  >
    link
  </a>
</footer>
`;
